subcommand/server-acl-init/rules.go

```diff
@@ -204,11 +205,25 @@ namespace "{{ .SyncConsulDestNS }}" {
 }
 
 func (c *Command) injectRules() (string, error) {
-	// The Connect injector only needs permissions to create namespaces.
+	// The Connect injector needs permissions to create namespaces and create/update service checks.
 	injectRulesTpl := `
 {{- if .EnableNamespaces }}
 operator = "write"
 {{- end }}
+{{- if .EnableHealthChecksController }}
+{{- if .EnableNamespaces }}
+namespace_prefix "" {
```

Comment:
I'm not 100% sure on the need for `namespace_prefix` for the non consul-ent case.
Once consul-ent comes into play I'm sure this will be required.
I'll do some more testing today to validate.

Reply:
I've confirmed it is not necessary for now and will be removing it with a commit before we merge.